systemd/ceph-disk@.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
What is the reason to create this service ?

Reply:
While I was reverting ceph-disk, this was one of the files referenced in ceph.spec.in the I got it back as well.